Kernel/Bus/PCI/Definitions.h

```diff
@@ -92,6 +99,20 @@ static constexpr u8 msix_table_bir_mask = 0x7;
 static constexpr u16 msix_table_offset_mask = 0xfff8;
 static constexpr u16 msix_control_enable = 0x8000;
 
+namespace AddressMetadata {
```

Comment:
Those definitions are OpenFirmware-specific, so maybe put them into a seperate file or namespace? As currently they look like generic PCI definitions.

Reply:
Good point, renamed the namespace to OpenFirmwareAddress